pycodestyle.py

```diff
@@ -131,7 +132,8 @@ def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
     'and', 'in', 'is', 'or'] +
     FUNCTION_RETURN_ANNOTATION_OP +
     ASSIGNMENT_EXPRESSION_OP)
-WHITESPACE = frozenset(' \t')
+NON_BREAKING_WHITESPACE = '\xa0'
```

Comment:
this constant isn't used anywhere but below -- inline it

Reply:
https://github.com/PyCQA/pycodestyle/pull/1035/commits/41545a21e9f29b5c9ab46d45b89e3ed099647886